testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -734,8 +733,12 @@ def __init__(self):
 
     def test_atomname_alignment(self):
         # Our PDBWriter adds some stuff up top, so line 1 happens at [4]
-        assert_(self.writtenstuff[4].startswith("ATOM      1  H5T GUA"))
-        assert_(self.writtenstuff[8].startswith("ATOM      5 H5'' GUA"))
+        refs = ("ATOM      1  H5T",
```

Comment:
so this is a nice **integration-test** but not a **unit-test** for `_deduce_PDB_atom_name`. Since the `_deduce_PDB_atom_name`  is a staticmethod you can test it by itself on a large amount of inputs quickly without needing to write files. Having both unit and integration test would be nice because it allows us to quickly check if the function gives the wrong result or if it is used incorrectly.